packages/dashboard/src/testing/testing-ground/testing-ground.tsx

```diff
@@ -43,6 +160,57 @@ export class TestingGround {
     this.stretchToFit = (e as any).target.checked;
   };
 
+  addAnnotation = (e: Event) => {
+    e.preventDefault();
+    const original = this.dashboardConfiguration;
+    const XANNOTATION: XAnnotation = {
+      color: 'red',
+      value: (e as any).target.value,
+      id: Date.now().toString(),
```

Comment:
Should we make a util for id generation?

Reply:
Could look into uuid for this